configs/icds-14lakh.yml

```diff
@@ -495,37 +520,70 @@ services:
     storage_scales_with_nodes: True
     process:
       cores_per_node: 16
-      ram_per_node: 32
+      ram_per_node: 48
     storage:
       group: 'VM_other'
-      static_baseline: 25GB
+      static_baseline: 100GB
       add_buffer: False
       override_estimation_buffer: 0
 
   redis:
     usage_capacity_per_node: 400000
     min_nodes: 3
     process:
-      cores_per_node: 16
-      ram_per_node: 32
+      cores_per_node: 4
+      ram_per_node: 64
+      ram_model:
+        - referenced_field: 'users'
+          unit_size: 50KB
+      ram_static_baseline: 33  # per node (assume only 50% ram is usable)
+    storage:
+      group: 'SAS'
+      data_models:
+        - referenced_field: 'users'
+          unit_size: 50KB
+
+  # potentially deploy redis as a cache for object storage if object storage
```

Comment:
As mentioned during the meeting today, I think we may want to do this with nginx instead